python/tests/artm/test_batches_utils.py

```diff
@@ -55,6 +55,7 @@ def test_func():
         model.fit_online(update_every=1, batch_vectorizer=in_memory_batch_vectorizer)
         assert len(model.score_tracker['perplexity'].value) == 14
 
+        del in_memory_batch_vectorizer
```

Comment:
Sorry, it was ambiguous question :)
I'd like to say 'Is it better to add comment before this line explaining what you want to test?' :)